internal/testing/cmdtest/cmdtest.go

```diff
@@ -269,57 +270,24 @@ func (tf *testFile) addCase(tc *testCase) []string {
 	return suffix
 }
 
-// Run runs Compare or Update, depending on the value of the argument.
-func (ts *TestSuite) Run(update bool) error {
+// Run runs CompareT or UpdateT, depending on the value of update.
```

Comment:
I think it's OK to remove remove Compare and Update and the *T versions, and just have a single unexported function for each. We don't have a use for them so far, and if we ever do we can add them back easily enough.

Reply:
The negative tests need the version that returns an error string. The best way I could think of to keep that was like this. I guess we could make that version unexported?

Reply:
Done. I just left a separate unexported function `compareReturningError` that's used in the test.